go.mod

```diff
@@ -1,24 +1,33 @@
-module github.com/dmwm/udp-collector
+module udp-collector
```

Comment:
There is no need to change module path even if your package is not in github. This is treated as simple string. Very often I create package with these commands:

```
go mod init github.com/..../package
go mod tidy
```
and I suggest that you do the same to rebuild the go.mod and go.sum.